packages/@okta/vuepress-theme-prose/util/coveo/endpoint.js

```diff
@@ -8,7 +8,7 @@ const COVEO_PIPELINE = "oktaproduction9ounvcxa";
 const COVEO_ENDPOINT = "https://platform.cloud.coveo.com/rest/search";
 
 const _getToken = () => {
-  const token = localStorage.getItem("coveo_token");
+  const token = localStorage.getItem("coveo_token")?.replace('"', '');
```

Comment:
does the build run through babel so that those code would be working for old browser like IE11.

Reply:
I was looking for this but let's go with the safer option

Reply:
Done